bigquery_etl/cli/query.py

```diff
@@ -520,30 +519,27 @@ def _backfill_query(
     run_checks,
 ):
     """Run a query backfill for a specific date."""
-    backfill_date_str = backfill_date.strftime("%Y-%m-%d")
-    if backfill_date_str in exclude:
-        click.echo(f"Skipping {query_file_path} backfill for run {backfill_date_str}")
-        return True
-
     project, dataset, table = extract_from_query_path(query_file_path)
     if destination_table is None:
         destination_table = f"{project}.{dataset}.{table}"
 
     # For partitioned tables, get the partition to write to the correct destination:
-    if (
-        partition := _get_partition(
-            backfill_date,
-            date_partition_parameter,
-            date_partition_offset,
-            partitioning_type,
-        )
-    ) is not None:
-        destination_table = f"{destination_table}${partition}"
+    if partitioning_type is not None:
```

Comment:
The separation of logic between `backfill` and `_backfill_query` isn't very obvious to me, but perhaps this could be moved to where you iterate over the `BackfillDateRange` object in `backfill`